src/views/certifications/views/[slug]/components/program-hero/index.tsx

```diff
@@ -0,0 +1,29 @@
+import classNames from 'classnames'
+// Shared under certifications
+import { CertificationsHero } from 'views/certifications/components'
+import { ProgramSlug } from 'views/certifications/types'
+// Styles
+import s from './program-hero.module.css'
+
+function ProgramHero({
+	heading,
+	description,
+	slug,
+}: {
```

Comment:
Can we extract this into an interface?

Reply:
Absolutely! ✅ extracted to an adjacent `types.ts` file 👍